composer.json

```diff
@@ -24,7 +24,8 @@
         "symfony/filesystem": "^4.4.0 || ^5.0.0 || ^6.0.0",
         "symfony/finder": "^4.4.0 || ^5.0.0 || ^6.0.0",
         "symfony/translation": "^4.4.0 || ^5.0.0 || ^6.0.0",
-        "symfony/validator": "^4.4.0 || ^5.0.0 || ^6.0.0"
+        "symfony/validator": "^4.4.0 || ^5.0.0 || ^6.0.0",
+        "ext-xml": "*"
```

Comment:
We have them in require-dev for now it seems.

Reply:
Does it make sense to add it to `require-de`v instead of `require`?

Reply:
If it is optional, so if there are ways to use the library without the need of it.

Reply:
Optional means that even in require mode there are some cases where it would not be needed.

Reply:
Why did you move it to require again?